src/components/Calendar/index.jsx

```diff
@@ -1,71 +1,63 @@
 'use client';
-import { useState } from 'react';
+import { useState, useEffect } from 'react';
+import CalendarMonthsHeader from '../CalendarMonthsHeader';
+import CalendarDayItem from '../CalendarDayItem';
+import { fillTableRows, now } from '@/utils/functions/caneldarContentGenerator';
```

Comment:
```suggestion
import { fillTableRows, now } from '@/utils/functions/calendarContentGenerator';
```

Fix the file name and the import